iron-validator-behavior.html

```diff
@@ -0,0 +1,62 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../iron-meta/iron-meta.html">
+
+<!--
+Use `Polymer.IronValidatorBehavior` to implement a custom input/form validator. Element instances
+implementing this behavior will be registered for use in elements that implement
+`Polymer.IronValidatableBehavior`.
+-->
+
+<script>
+
+  Polymer.IronValidatorBehavior = {
+
+    properties: {
+
+      /**
+       * Namespace for this validator.
+       */
+      validatorType: {
+        type: String,
+        value: 'validator'
+      },
+
+      /**
+       * Name for this validator, used by `Polymer.IronValidatableBehavior` to lookup this element.
+       */
+      validatorName: {
+        type: String,
+        value: function() {
+          return this.is;
+        }
+      }
+
+    },
+
+    ready: function() {
+      new Polymer.IronMeta({type: this.validatorType, key: this.validatorName, value: this});
+    },
+
+    /**
+     * Implement custom validation logic in this function.
+     *
+     * @method validate
+     */
+    /*
+    validate: function(values) {
```

Comment:
How come this is commented out? Didn't they make it so that we can do abstract methods in behaviors now?